app/gui/src/main/scala/org/bitcoins/gui/dlc/dialog/CreateDLCOfferDialog.scala

```diff
@@ -22,10 +22,18 @@ import java.time.ZoneOffset
 import scala.collection._
 import scala.util.{Failure, Success, Try}
 
-class CreateDLCOfferDialog extends Logging {
+class CreateDLCOfferDialog extends Logging with CliCommandProducer {
 
-  def showAndWait(parentWindow: Window): Option[CreateDLCOffer] = {
-    val dialog = new Dialog[Option[CreateDLCOffer]]() {
+  def getCliCommand() = {
+    createDLCOffer()
+  }
+
+  private var dialog: Dialog[Option[CreateDLCOffer]] = null
```

Comment:
Don't use `null` ever. It's the billion dollar mistake with java. 

With Scala, we can use `Option` to represent `null`. Here is the diff for fixing this file:

https://github.com/Christewart/bitcoin-s-core/commit/9790618ee6acabc63f6b76a1b688a9d29e9914c1